test/parallel/test-crypto-sign-verify.js

```diff
@@ -15,6 +15,40 @@ const certPem = fixtures.readSync('test_cert.pem', 'ascii');
 const keyPem = fixtures.readSync('test_key.pem', 'ascii');
 const modSize = 1024;
 
+{
+  const Sign = crypto.Sign;
+  const instance = Sign('SHA256');
+  assert.ok(instance instanceof Sign, 'call sign constructor without new');
+}
+
+{
+  const Verify = crypto.Verify;
+  const instance = Verify('SHA256');
+  assert.ok(instance instanceof Verify, 'call sign constructor without new');
```

Comment:
sign -> verify

Reply:
Ah, I made a mistake.
Thank you for your review.